### ISSUE TITLE
Incorrect window size on the "submit work" page

### ISSUE THREAD

**Issue by polikosi** [Joystream/pioneer#3201]
## Context

Incorrect window size

the bottom of the modal is showing beneath screen fold
![Screenshot_102](https://user-images.githubusercontent.com/80429191/172436638-362f198a-e012-4b09-977a-5e3a6b2e4d1c.png)

## Scope

- [ ] Change the styling on the modal to support good centering and sizing
⚠️ pls do together with - https://github.com/joystream/pioneer/issues/3025
and - #2976

**Comment by freakstatic** [comment]
@singulart please assign

**Comment by singulart** [comment]
Should be fixed by https://github.com/Joystream/pioneer/issues/3025

**Comment by polikosi** [comment]
verified on https://pioneer-2.vercel.app/#/profile . Not fixed
![Screenshot_129](https://user-images.githubusercontent.com/80429191/175525906-275f6156-e62f-4fcf-8d1c-28bbb7cb57e7.png)

verified on https://dao-git-staging-joystream.vercel.app/#/profile . Not fixed
![Screenshot_128](https://user-images.githubusercontent.com/80429191/175526027-9daca958-3a6f-4954-8889-e4415d0ab49f.png)

**Comment by thesan** [comment]
After #3322 on firefox modals the `<ModalGlass />` dimension looks good but `<ModalWrap />` is now extra short:
![Screen Shot 2022-07-15 at 13 16 19](https://user-images.githubusercontent.com/6571453/179213613-7e92cb9f-008c-4c2f-98df-e4f1fa782a86.png)

**Comment by dmtrjsg** [comment]
@freakstatic pls see last comment from @thesan. Feel free to address it when the time is good.

**Comment by WRadoslaw** [comment]
It should be just the matter of removing `max-height` from `ModalWrap`. I just realised that `ModalGlass` has a `padding-bottom` that is acting as `max-height`. @freakstatic

**Comment by freakstatic** [comment]
> After #3322 on firefox modals the `<ModalGlass />` dimension looks good but `<ModalWrap />` is now extra short: ![Screen Shot 2022-07-15 at 13 16 19](https://user-images.githubusercontent.com/6571453/179213613-7e92cb9f-008c-4c2f-98df-e4f1fa782a86.png)

thanks for reporting!
I did a new PR here to address that:
https://github.com/Joystream/pioneer/pull/3367
